src/binary-reader.cc

```diff
@@ -1605,6 +1605,9 @@ static void read_custom_section(Context* ctx, uint32_t section_size) {
       uint32_t subsection_size;
       in_u32_leb128(ctx, &name_type, "name type");
       in_u32_leb128(ctx, &subsection_size, "subsection size");
+      size_t subsection_end = ctx->offset + subsection_size;
+      if (subsection_end > ctx->read_end)
+        RAISE_ERROR("invalid sub-section size: extends past end");
```

Comment:
I think the plan was for it to become "utf8-name" or "utf8-names"; I don't know whether that was retracted or not (@sunfish or @lukewagner might). Waiting for changes to make it into the spec requires some patience (the "official" spec still says that the version is 0xd...)